internal/telemetry/exporter.go

```diff
@@ -81,6 +81,8 @@ type NICResourceCounts struct {
 	Ingresses int64
 	// IngressClasses is the number of Ingress Classes.
 	IngressClasses int64
+	// Policies is the number of policy resources managed by the NGINX Ingress Controller
```

Comment:
```suggestion
	// Policies is the number of policy resources managed by NGINX Ingress Controller
```

Reply:
Just a heads up, if you change this command, you will need to run the `make telemetry-schema` command. This will make sure the [data.avdl](https://github.com/nginxinc/kubernetes-ingress/blob/c0864576bff37d6d4e25d882b97e3676d3cf2678/internal/telemetry/data.avdl#L60-L61) file is also updated.